perf/h.js

```diff
@@ -0,0 +1,26 @@
+import bench from 'nanobench'
+import { h } from '../src/h'
+
+bench("vnode with no child 1M times", b => {
+  b.start()
+  for (let i = 0; i < 10000000; i++) {
```

Comment:
If we want to be super precocinous here is how `jsperf` & cie do it: https://gist.github.com/ngryman/fd9bcd96910409f91af00d9fd4178a10